tools/yulPhaser/FitnessMetrics.cpp

```diff
@@ -62,15 +61,15 @@ size_t ProgramSize::evaluate(Chromosome const& _chromosome)
 
 size_t RelativeProgramSize::evaluate(Chromosome const& _chromosome)
 {
-	double const scalingFactor = pow(10, m_fixedPointPrecision);
+	double const scalingFactor = std::pow(10, static_cast<double>(m_fixedPointPrecision));
```

Comment:
Hi @cameel 
I removed this cast. I encountered a warning yesterday that said the implicit conversion from `size_t` to `double` would be an issue because of the signature of `double pow(double, double)`. After I removed this and recompiled it today, this warning is gone.